src/AdaptiveCurveIrm.sol

```diff
@@ -54,32 +42,10 @@ contract AdaptiveCurveIrm is IAdaptiveCurveIrm {
 
     /// @notice Constructor.
     /// @param morpho The address of Morpho.
-    /// @param curveSteepness The curve steepness (scaled by WAD).
-    /// @param adjustmentSpeed The adjustment speed (scaled by WAD).
-    /// @param targetUtilization The target utilization (scaled by WAD).
-    /// @param initialRateAtTarget The initial rate at target (scaled by WAD).
-    constructor(
-        address morpho,
-        int256 curveSteepness,
-        int256 adjustmentSpeed,
-        int256 targetUtilization,
-        int256 initialRateAtTarget
-    ) {
+    constructor(address morpho) {
         require(morpho != address(0), ErrorsLib.ZERO_ADDRESS);
```

Comment:
Like suggested by this comment https://github.com/morpho-org/morpho-blue-irm/pull/103/files#r1423108397 it would be good to have those sanity checks added (as an additional check) at deployment level so you always know that the deployed IRM is conforming the previous established ranges.

Reply:
We added a test for this, I think that's enough